webpack/components/extensions/HostDetails/Tabs/ErrataTab.js

```diff
@@ -0,0 +1,160 @@
+import TableWrapper from '../../../../components/Table/TableWrapper';
+import { getInstallableErrata } from '../HostErratum/HostErratumActions';
+import { selectHostErrataStatus } from '../HostErratum/HostErratumSelectors';
+import { HOST_ERRATUM_KEY } from '../HostErratum/HostErratumConstants';
+
+export const ErrataType = ({ type }) => {
+  let ErrataIcon;
+  let label;
+  switch (type) {
+    case 'security':
+      label = __('Security');
+      ErrataIcon = SecurityIcon;
+      break;
+    case 'recommended':
+    case 'bugfix':
+      label = __('Bugfix');
+      ErrataIcon = BugIcon;
+      break;
+    case 'enhancement':
+    case 'optional':
+      label = __('Enhancement');
+      ErrataIcon = EnhancementIcon;
+      break;
+    default:
+      label = '';
+  }
+  if (!ErrataIcon) {
+    return null;
+  }
+
+  return (
+    <TableText wrapModifier="nowrap">
+      <ErrataIcon title={label} /> {label}
+    </TableText>
+  );
+};
+
+ErrataType.propTypes = {
+  type: PropTypes.string.isRequired,
+};
+
+export const ErrataSeverity = ({ severity }) => {
+  let color;
+  let label;
+
+  switch (severity) {
+    case 'Moderate':
+      color = 'yellow';
+      label = __('Moderate');
+      break;
+    case 'Important':
+      color = 'orange';
+      label = __('Important');
+      break;
+    case 'Critical':
+      color = 'red';
+      label = __('Critical');
+      break;
+    default:
+      label = __('N/A');
+  }
+  return <TableText wrapModifier="nowrap">{color && <SecurityIcon color={color} title={label} />} {label} </TableText>;
+};
+
+ErrataSeverity.propTypes = {
+  severity: PropTypes.string.isRequired,
+};
+
+export const ErrataTab = () => {
+  const hostDetails = useSelector(state => selectAPIResponse(state, 'HOST_DETAILS'));
+  const { id: hostId } = hostDetails;
+
+  const emptyContentTitle = __('This host currently does not have any installable errata for this content view.');
+  const emptyContentBody = __('Installable Errata will appear here when available.'); // needs link
+  const emptySearchTitle = __('No matching Installable Errata found');
+  const emptySearchBody = __('Try changing your search settings.');
+  const [searchQuery, updateSearchQuery] = useState('');
+  const columnHeaders = [
+    __('Errata'),
+    __('Type'),
+    __('Severity'),
+    __('Synopsis'),
+    __('Published Date'),
+  ];
+
+  const fetchItems = useCallback(
+    params => getInstallableErrata(hostId, params),
+    [hostId],
+  );
+
+  const response = useSelector(state => selectAPIResponse(state, HOST_ERRATUM_KEY));
+  const { results, ...metadata } = response;
+  const status = useSelector(state => selectHostErrataStatus(state));
+
+  return (
+    <TableWrapper
+      {...{
+          metadata,
+          emptyContentTitle,
+          emptyContentBody,
+          emptySearchTitle,
+          emptySearchBody,
+          searchQuery,
+          updateSearchQuery,
+          status,
+          }}
+      fetchItems={fetchItems}
```

Comment:
Should add the disabled "Apply" button here, via an `actionButtons` prop.  Also add the action menu and disabled action items.

Reply:
updated